stocks@infinicode.de/helpers/fetch.js

```diff
@@ -10,7 +10,7 @@ const Response = class {
 
     if (message) {
       this.headers = message.response_headers
-      this.url = message.get_uri().to_string()
+      this.url = message.get_uri().to_string(true)
```

Comment:
I'm not sure about this change. It was already changed in the previous [commit](https://github.com/cinatic/stocks-extension/commit/55490bcda8960c11c19ceec1ca92102e047fd791).

But it was the only change to let the extension work with GS42.

If this signature is not compatible with the GS43, maybe we should consider wrapping it into a `try/catch` block and falling back to one of the versions in `catch`. Then we can add the GS42 as compatible with the latest extension version.

What do you think?